CHANGELOG.md

```diff
@@ -46,8 +46,9 @@
  * If a payment is sent, creating an outbound HTLC and sending it to our
    counterparty (implying the `ChannelMonitor` was persisted on disk), but the
    `ChannelManager` was not persisted prior to shutdown/crash, no
-   `Event::PaymentPathFailed` event will be generated if the HTLC is eventually
-   failed on chain (#1104).
+   `Event::PaymentPathFailed` event was be generated if the HTLC is eventually
```

Comment:
s/was be/was
s/is/

Reply:
I tweaked the case to be past-tense more consistently.